spec/classes/api_spec.rb

```diff
@@ -1,7 +1,9 @@
 require 'spec_helper'
 
 describe 'sensu::api', :type => :class do
-  on_supported_os.each do |os, facts|
+  on_supported_os({
+    supported_os: [{ 'operatingsystem' => 'RedHat', 'operatingsystemrelease' => ['7'] }]
```

Comment:
Why did this change?

Reply:
To speed up unit tests.  This and several other unit test files don't change based on OS tested, and we already did this pattern on a few other files before this pull request.  The classes that actually change based on OS facts still test all OSes defined in metadata.json.

Reply:
Please add a comment with each that explains why this is happening.

Reply:
Comment added